externalpolicy/text_language/text_language_test.go

```diff
@@ -101,7 +101,7 @@ func Test_Validate_Failed3(t *testing.T) {
 		t.Errorf("Validation should fail but did not, err: %v", err)
 	} else if err == nil {
 		t.Errorf("Validation should fail and return err, but didn't")
-	} else if err.Error() != "Logical operator load is not valid, expecting AND, OR, &&, ||" {
+	} else if err.Error() != "No control operator found:  load && certification == USDA" {
```

Comment:
make check has 3 failures, the first one is here

Reply:
Sorry changed error messages after final make check run. Fixed